compress.go

```diff
@@ -48,7 +57,7 @@ func CompressHandler(h http.Handler) http.Handler {
 				w.Header().Set("Content-Encoding", "gzip")
 				w.Header().Add("Vary", "Accept-Encoding")
 
-				gw := gzip.NewWriter(w)
+				gw, _ := gzip.NewWriterLevel(w, level)
```

Comment:
Instead of ignoring the error, it'd be better to instead check the `level` is within acceptable bounds. If not, we should just set `gzip.DefaultCompression` as the level.